src/views/usages/UsagesTree/UsagesTreeProvider.ts

```diff
@@ -119,4 +119,26 @@ export class UsagesTreeProvider
 
     return []
   }
+
+  // In order to use TreeView.reveal method, we need to implement getParent method
+  getParent(element: CodeUsageNode): vscode.ProviderResult<CodeUsageNode> {
+    return element
+  }
+
+  findElementByKey(key: string): CodeUsageNode | undefined {
```

Comment:
it seems like it would make sense to pass the folder context to this command rather than leaving it to determine which file was clicked